scripts/bootstrapData.ts

```diff
@@ -1,16 +1,19 @@
+import "dotenv/config"
```

Comment:
move dotenv stuff into the file so `-r` isn't needed on cmd line and can be more self contained